src/drawing/DrawingThread.js

```diff
@@ -121,6 +121,57 @@ class DrawingThread extends AnnotationThread {
         this.clearBoundary();
     }
 
+    /**
+     * Binds DOM event listeners for drawing new thread using
+     * mode specific location getter
+     *
+     * @protected
+     * @param {Function} locationFunction - Location getter method
+     * @return {void}
+     */
+    bindDrawingListeners(locationFunction) {
```

Comment:
Binding/unbinding of these listeners was moved from inside the drawingmodecontroller